packages/core/src/layout/ErrorBoundary/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-console */
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { ErrorBoundary } from './ErrorBoundary';
+import { renderInTestApp } from '@backstage/test-utils';
+
+type BombProps = {
+  shouldThrow?: boolean;
+  children?: React.ReactNode;
+};
+
+const Bomb = ({ shouldThrow }: BombProps) => {
+  if (shouldThrow) {
+    throw new Error('Bomb');
+  } else {
+    return <p>Working Component</p>;
+  }
+};
+
+const spiedConsole = jest.spyOn(console, 'error');
```

Comment:
Use `withLogCollector` from `@backstage/test-utils`!

Reply:
Thank you for suggesting this, I was not aware of withLogCollector.